pkg/v1/tkg/client/describe_cluster.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/runtime"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
IIUC, you could use v1beta1 APIs to query v1alpha3 clusters, since there is a conversion webhook.

Reply:
Since TKGS is using CAPI v1alpha3  and TKGm v1beta1 clusters can be queries using v1alpha3 API, for now we can use CAPI v1alpha3 for `DescribeCluster` API which would work for TKGS and TKGm.

Reply:
Just made the change for describe_cluster to use v1alpha3 API instead of v1beta1 and verified the change against Azure.

Reply:
Created new issue to track this work - https://github.com/vmware-tanzu/tanzu-framework/issues/1063

Reply:
Resolving this conversation for now and take this separately.